source/admin_manual/list_of_events.rst

```diff
@@ -974,6 +974,31 @@ POP3
    :field remote_port @added;v2.4.0,v3.0.0: POP3 connection's remote (client) port.
 
 
+.. dovecot_event:field_group:: pop3_command
+   :inherit: pop3_client
+
+   :field cmd_name @added;v2.4.0,v3.0.0: POP3 command name uppercased (e.g. ``UIDL``).
+   :field cmd_args @added;v2.4.0,v3.0.0: POP3 command's full parameters (e.g. ``1 1``).
```

Comment:
:added: for this whole pop3_command and remove `@added;v2.4.0,v3.0.0`?

Also, these field_groups don't actually seem to be working (even in main). They're invisible.

Reply:
I can't reproduce any issues with field_groups (building today with main).  What are you seeing?
field_groups are NOT viewable blocks.  They define groups of parameters that can be inherited by, e.g., events.  For example, field_group `imap_client` is inherited by imap_client_hibernated - and I see user, session, local_ip, local_part, remote_ip, and remote_part in the attribute list.

Reply:
Oh, didn't realize they worked that way. And looks like there is also no `:added:` for the group.